bin/mirgecompare.py

```diff
@@ -1,8 +1,17 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
```

Comment:
Most things installed through emirge seem to have `#!/usr/bin/env python` but I'm agnostic on this.

Reply:
Fair. Reverted for consistency.